shotover-proxy/benches/cassandra_benches.rs

```diff
@@ -1,3 +1,66 @@
-fn main() {
-    println!("Hello world");
+use cassandra_cpp::{stmt, Session};
+use criterion::{criterion_group, criterion_main, Criterion};
+use test_helpers::docker_compose::DockerCompose;
+
+#[path = "../tests/helpers/mod.rs"]
+mod helpers;
+use helpers::ShotoverManager;
+
+fn cassandra(c: &mut Criterion) {
+    let mut group = c.benchmark_group("cassandra");
+    group.throughput(criterion::Throughput::Elements(1));
+    group.noise_threshold(0.2);
+
+    let statement =
+        stmt!("INSERT INTO benchmark_keyspace.table_1 (id, x, name) VALUES (1, 11, 'foo');");
+
+    {
+        group.bench_with_input(
+            "passthrough",
+            || {
+                let compose =
+                    DockerCompose::new("examples/cassandra-passthrough/docker-compose.yml")
+                        .wait_for_n_t("Startup complete", 1, 90);
+                let shotover_manager = ShotoverManager::from_topology_file(
+                    "examples/cassandra-passthrough/topology.yaml",
+                );
+                BenchResources::new(shotover_manager, compose)
+            },
+            |b, state| {
+                b.iter(|| {
+                    state.connection.execute(&statement).wait().unwrap();
+                })
+            },
+        );
+    }
+}
+
+criterion_group!(benches, cassandra);
+criterion_main!(benches);
+
+pub struct BenchResources {
+    _compose: DockerCompose,
+    _shotover_manager: ShotoverManager,
+    connection: Session,
+}
+
+impl BenchResources {
+    #[allow(unused)]
+    pub fn new(shotover_manager: ShotoverManager, compose: DockerCompose) -> Self {
+        let connection = shotover_manager.cassandra_connection("127.0.0.1", 9043);
```

Comment:
we are benching directly against the DB without shotover!
Shotover is currently on port 9042.

Reply:
Oops, fixed!